### PR TITLE
Document new query capabilities

### PR DESCRIPTION
**Pull request by leondutoit**
* new pysquril features for survey and apps
* update apps with aggregations

### CODE REVIEW

File: integration/apps-api.md [hunk 1]
```diff
@@ -206,6 +211,40 @@ range=1.2
 ```
 returns up to two records of the dataset, starting with the second record.
 
+#### Aggregation
+
+Clients can perform the following aggregation functions on data:
+
+* `count` - the number of entries in a selection
+* `avg` - geomertic average
```
Comment (uio-torjus) [inline integration/apps-api.md:219]: typo

File: integration/apps-api.md [hunk 2]
```diff
@@ -206,6 +211,40 @@ range=1.2
...
+
+For example, to get the number of table entries, along with the timestamp of the most recent entry:
+
+```txt
+?select=count(*),max_ts(metaData.timestamp)
+```
+
+#### Broadcasting queries
+
+Clients can apply queries to mulitple endpoints at the same time, by using fuzzy matching on endpoint names. Some examples follow.
```
Comment (uio-torjus) [inline integration/apps-api.md:234]: typo

File: integration/survey-api.md [hunk 3]
```diff
@@ -361,14 +367,27 @@ For example, to get the number of table entries, along with the timestamp of the
 ?select=count(*),max_ts(metaData.timestamp)
 ```
 
-Aggregate queries can be broadcast over all tables as such:
+#### Broadcasting queries
+
+Clients can apply queries to mulitple endpoints at the same time, by using fuzzy matching on endpoint names. Some examples follow.
```
Comment (uio-torjus) [inline integration/survey-api.md:372]: typo

**Review by uio-torjus — APPROVED** [review]

**Review by egiltane — APPROVED** [review]

File: integration/survey-api.md (hunk 3 above)
Comment (leondutoit) [reply integration/survey-api.md]: ty, fixed.

File: integration/apps-api.md (hunk 2 above)
Comment (leondutoit) [reply integration/apps-api.md]: fixed

**Pull request merged**